src/firmware_new/boards/frankie_v1/io/BUILD

```diff
@@ -31,6 +31,13 @@ cc_library(
     ],
 )
 
+cc_library(
+    name = "infineon_tle5009_e1000_sensor",
```

Comment:
Perhaps suffix the target (and the files) with `_angle_sensor`?

Reply:
Done